hoomd/md/pair.py

```diff
@@ -1414,72 +1176,57 @@ class moliere(pair):
                                 = & 0 & r > r_{\mathrm{cut}} \\
         \end{eqnarray*}
 
-    See :py:class:`_Pair` for details on how forces are calculated and the available energy shifting and smoothing modes.
-    Use ``coeff.set`` to set potential coefficients.
-
-    The following coefficients must be set per unique pair of particle types:
+    Where each parameter is defined as:
 
     - :math:`Z_i` - *Z_i* - Atomic number of species i (unitless)
     - :math:`Z_j` - *Z_j* - Atomic number of species j (unitless)
     - :math:`e` - *elementary_charge* - The elementary charge (in charge units)
     - :math:`a_0` - *a_0* - The Bohr radius (in distance units)
```

Comment:
`a_0` isn't actually used in the above expression. Put the definition of `a_F` here with ending with `, where *a_0* is the Bohr radius (in distance units)`